oic.r.roles.raml

```diff
@@ -2,7 +2,7 @@
 title: OICSecurityRolesResource
 version: v1.0-20170323
 schemas:
-  - Cred: !include schemas/oic.r.cred.json
+  - Roles: !include schemas/oic.r.roles.json
```

Comment:
I have the same understanding of this resource. In that case this MR should be accepted